Tutorials/Editors-Manual/Working-with-Content/Rich-Text-Editor/index.md

```diff
@@ -221,7 +221,9 @@ The macros available to you differ from system to system so please see your syst
 
 ![macroPicker.jpg](images/Macro-picker-button.png)
 
-![macroPicker.jpg](images/Insert-macro.png)
+![macroPicker.jpg](images/Select-a-macro.png)
+
+*note that if there is no macros to chose from it will not look like this- this is from the starter kit package*
```

Comment:
Thanks for getting the image updated 😄 

About the note here .. Shouldn't we use the note formatting instead?
```suggestion
:::note
If there is no macros to choose from it will not look like this. What you see above, is how it looks when the Starter Kit package is installed.
:::
```